packages/osd-plugin-generator/src/integration_tests/generate_plugin.test.ts

```diff
@@ -32,7 +32,8 @@ import Path from 'path';
 
 import del from 'del';
 import execa from 'execa';
-import { REPO_ROOT, standardize, createAbsolutePathSerializer } from '@osd/dev-utils';
+import { standardize, REPO_ROOT } from '@osd/cross-platform';
```

Comment:
had to delete the `standardize` here since it causes the bootstrap to fail because it is unused. 
![image (1)](https://user-images.githubusercontent.com/43937633/199143098-8bd6b3c1-1190-4020-823c-dcb4dc02c4ca.png)

Reply:
thanks

Reply:
Thanks. Fixed.